### PR TITLE
Mention the published lightspeed image on quay.io in readme

### PR DESCRIPTION
**Pull request by bparees**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [x] Documentation Update

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by bparees** [comment]
/assign @vishnuchalla 

ptal.  There's a lot more we could do here, given that now we have 3 ways to run:

1) run the python code directly
2) use the kustomize resources to deploy on a cluster (with or w/o first building your own image)
3) run the image locally (with or w/o first building your own image)

and for each of them the process of actually interacting w/ the server is slightly different (especially when you run on cluster since then you need to pass a service account token that has RBAC permission to interact w/ the service)

File: README.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ By default this interface will ask the OLS server to retain and use your convers
 ### In Cluster Deployment
 Deploying OLS on an openshift cluster is fairly easy with the configuration files we have in [config](./config) folder.
 
-Initial step would be to build and publish our own image to an image registry. Example:
+You can use the existing image built from the latest code: quay.io/openshift/lightspeed-service-api:latest
+
+If you need to build your own image, you can use the following commands:
 
 ```
 podman build -f Containerfile -t=quay.io/<your-repo>/ols:latest .
```
Comment (vishnuchalla) [inline README.md:77]: I think we need to keep this same as line #83 for consistency.

Comment (bparees) [reply README.md:77]: i meant to change it to:
export OLS_IMAGE=<image-pullspec>

and i'm going to change the others also (no reason to assume they are using quay)

Comment (bparees) [reply README.md]: updated

**Pull request merged**